src/lib.rs

```diff
@@ -23,24 +30,30 @@ pub struct GroveDb {
 impl GroveDb {
     pub fn new() -> Result<Self, Error> {
         let subtrees = vec![
-            Merk::open("./test1.db")?,
-            Merk::open("./test2.db")?,
-            Merk::open("./test3.db")?,
-            Merk::open("./test4.db")?,
-            Merk::open("./test5.db")?,
-            Merk::open("./test6.db")?,
+            Merk::open(Path::new(SUBTREES_DIR).join("test1.db"))?,
```

Comment:
All trees need to be stored in one database

Reply:
Yeah, it will be a scope of a next task

Reply:
Actually, done in this PR, please take a look